alonzo/formal-spec/utxo.tex

```diff
@@ -659,7 +663,7 @@ \subsection{The UTXOS transition system}
       \var{txb}\leteq\txbody{tx} &
       \fun{ininterval}~\var{slot}~(\fun{txvldt}~{tx}) &
       \hldiff{\var{(\wcard, i_f)}\leteq\fun{txvldt}~{tx}} \\~\\
-      \hldiff{\fun{txrdmrs}~\var{tx}\neq \Nothing ~\Rightarrow~ \fun{epochInfoSlotToUTCTime}~\mathsf{epochInfo}~\mathsf{systemTime}~i_f \neq \Nothing} \\
+      \hldiff{\Nothing \notin \{\fun{txrdmrs}~\var{tx}, i_f\} ~\Rightarrow~ \fun{epochInfoSlotToUTCTime}~\mathsf{epochInfo}~\mathsf{systemTime}~i_f \neq \Nothing} \\
```

Comment:
\notin?

Reply:
It's a compact way of saying that both of those aren't Nothing. So this modifies the line to only check the conversion if `i_f` isn't nothing as well

Reply:
right, didnt notice it was \notin ... a set